decidim-challenges.gemspec

```diff
@@ -10,14 +10,13 @@ Gem::Specification.new do |s|
   s.email = ["199462+tramuntanal@users.noreply.github.com"]
   s.license = "AGPL-3.0"
   s.homepage = "https://github.com/decidim/decidim-module-challenges"
-  s.required_ruby_version = ">= 3.0.6"
+  s.required_ruby_version = ">= 3.1.1"
 
   s.name = "decidim-challenges"
   s.summary = "A decidim challenges module"
   s.description = "Articulates the collective action of diverse actors in order to address shared challenges and the problems that derive from them across the territory."
 
   s.files = Dir["{app,config,db,lib}/**/*", "LICENSE-AGPLv3.txt", "Rakefile", "README.md"]
-  s.test_files = Dir["spec/**/*"]
```

Comment:
why remove?

Reply:
Because rubocop

```
decidim-challenges.gemspec:20:3: W: [Corrected] Gemspec/DeprecatedAttributeAssignment: Do not set test_files in gemspec.
  s.test_files = Dir["spec/**/*"]
```